guides/gatherers.md

```diff
@@ -209,6 +219,15 @@ Example output (in Rhai):
 
 // node_0_ring0addr
 20;
+
```

Comment:
Make me a favor. It was not in your PR, but change the line above (`// node_0_ring0addr` to have `"10.80.1.11"` value below). It was wrong from before

Reply:
Oh wow, totally missed that 👀 thanks

Reply:
Why does it add parenthesis there??
It happened the same with `udpu`. In that case was due the prettier. Could you remove them?

Reply:
I hope it doesn't affect you @fabriziosestito

Reply:
Removed 👍 I looked through the rest of the file and didn't find other instances of it happening